### DISCUSSION TITLE
bootswatch

### DISCUSSION THREAD

**Discussion by alezunino** [alshedivat/al-folio#1711] [Ideas]
Hi,
Despite al-folio provides some mechanisms for custom colors/themes, they are limited to changing colors in _sass/_variables.scss. Recently, I have been trying to replace bootstrap by [bootswatch](https://bootswatch.com/) in al-folio, but since _base.scss defines "color" attributes in many places, it is not straightforward. Have you considered integrating bootswatch into al-folio?

**Comment by george-gca** [comment]
I think `bootswatch` change too much stuff, so integrating it to enable these readily available themes would be too much of an effort to make most of them look good. The idea of the template is to create something that just works out of the box, but customizing it to your liking shouldn't be too much trouble assuming you know what you are doing.

As for the colors attributes, most of them are defined in [_variables.scss](https://github.com/alshedivat/al-folio/blob/master/_sass/_variables.scss). Here is more something like `color name: hex`. In [_themes.scss](https://github.com/alshedivat/al-folio/blob/master/_sass/_themes.scss) the colors are "mostly" defined as `usage: color name`.

But feel free to send a PR, maybe integrating `bootswatch` is not as hellish as I am presuming.
